src/campaign.cairo

```diff
@@ -27,6 +27,9 @@ mod TokengiverCampaign {
     use openzeppelin::access::ownable::OwnableComponent;
     use openzeppelin::upgrades::UpgradeableComponent;
     use openzeppelin::upgrades::interface::IUpgradeable;
+    use token_bound_accounts::interfaces::ILockable::{
```

Comment:
So you only import the component you are interested in,

nice 
Good Work

Reply:
Thanks big chief.